Draw.cpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include "Engine.h"
+
+using namespace sf;
+
+
+
+void Engine::draw()
+{
+
+	//Clear the last frame
+	m_Window.clear(Color::Cyan);
+	
+
+	//Draw the background
+	m_Window.draw(m_BackgroundSprite);
+
+	
+	//Spawn and draw the cupcakes
+	cupcake.update(m_Window);

```

Comment:
Usually when making games, `update` is done separate from the `draw`. Like `Engine::update`.